src/index.js

```diff
@@ -215,10 +235,41 @@ class ExtractTextPlugin {
 
             const file = (isFunction(filename)) ? filename(getPath) : getPath(filename);
 
-            compilation.assets[file] = source;
-            chunk.files.push(file);
+            let preventOutput = false;
+            this.options.merge.forEach((mergeChunk) => {
+              if (mergeChunk.test.test(file)) {
+                if (!this.filesToMerge[mergeChunk.filename]) {
+                  this.filesToMerge[mergeChunk.filename] = [];
+                }
+                this.filesToMerge[mergeChunk.filename].push(extractedChunk);
+                preventOutput = preventOutput || (mergeChunk.preventOriginalOutput !== false);
```

Comment:
Can we find a better name (less verbose) then `preventOriginalOutput` => `prevent || original || preventOriginal || ?` ?

Reply:
sure, I was thinking also about that :P

I will include the tests, but wanted to make sure it wasn't going to become a dropped feature :)

Reply:
😛  Well we definitely need to check for ordering issues && multi entry setups and it needs approval from other maintainers aswell. But the feature itself is a nice improvement imho